src/features/header/ui.jsx

```diff
@@ -14,7 +14,7 @@ export function Header() {
                 <h1 className="hero__title">{siteConfig.title}</h1>
                 <p className="hero__subtitle">{translate({ id: "features.hero.tagline" })}</p>
                 <div className="button-group">
-                    <Link className="button button--primary button--lg" to="/docs/intro">
+                    <Link className="button button--primary button--lg" to="/docs">
```

Comment:
> 1. The actual textual content is too far down the website.

Mostly agree, but what if we do next:
- GET_STARTED button => `/docs/get-started/overview`
- Docs navlink => `/docs`

On my opinion we solve your remarks:
- we omit redundant IndexPages navigating and navcards clicking
- we go to "readable text" at one click 🎊

Reply:
Navcards also could be simplified (we have #445  issue)

Reply:
My concern with this proposal is that this will create a page in the flow that feels "preceding" and that would make me afraid as a reader of missing out.

What I mean is:
If the flow is `Page 1 => Page 2 => Page 3 => ...`, but the link in the landing page sends me to page 3, I will be okay, I will start reading and everything will be fine. That is, until I eventually accidentally stumble onto pages 1 and 2 and start thinking "oh shi-, I think I didn't start at the beginning".

If some part of the methodology doesn't make sense to me, I will be more likely to blame it on myself with the thoughts along the lines of "well I should've read from the beginning, there are probably some ground things and base concepts that I didn't grasp". That way, we might get less users complaining about the docs even in cases when the docs are the problem. 

My personal ideal is not to go to readable text at one click, and it's never really about the clicks, as Steve Krug says in an UX design classic "Don't Make Me Think". What matters is that in our case, with every click comes a decision:
* [on /docs] I see a lot of nav cards, I'll click the first but I need to remember to check out the rest
* [on /docs/get-started] Oh hell, there are more choices here... Clicking the first one, gotta remember...

Again, what makes Gatsby so nice is that they have very few options and the every option gets you started right away. So I suggest to keep the landing page link to lead to `/docs`, but change the links of the nav cards in the `/docs`

Reply:
Lets discuss on next call

Reply:
fixed